### PR TITLE
912 new UI application status

### PR DESCRIPTION
**Pull request by tarun9142**
Targets #912

### Description
*updated the UI for application status tabs 

These changes updates the job application status bar UI  on recruitment page.
app status
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,66 +0,0 @@
-APP_NAME=Laravel
```
Comment (GauravGusain98) [inline .env.example:1]: You don't have to remove this file. Just make a copy of it and rename that as `.env`

File: Modules/HR/Resources/views/application/index.blade.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <br>
     <div class="row">
         <div class="col-md-6">
-            <h1>Applications</h1>
+            <h1>Applications dsgrggfdsg</h1>
```
Comment (GauravGusain98) [inline Modules/HR/Resources/views/application/index.blade.php:10]: What's this? Please fix it.

File: package.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
         "vue-template-compiler": "^2.6.11"
     },
     "dependencies": {
-        "axios": "^0.19.2",
+        "axios": "^0.21.1",
```
Comment (GauravGusain98) [inline package.json:36]: Why you updated this package. If the update is not required for the feature, then, please don't add package files to git

File: resources/views/hr/application/index.blade.php [hunk 4]
```diff
@@ -46,102 +46,145 @@
...
-                <a class="nav-item nav-link d-flex align-items-center {{ $status ? 'text-info' : 'active bg-info text-white' }}"
-                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}}> <i
-                    class="fa fa-clipboard"></i>&nbsp;
+    <div class="d-flex bg-white status-icons ">
+        <ul class="nav mb-2 d-flex justify-content-between">
+            <li class="nav-item mx-3">
+                <a class="nav-item nav-link d-flex flex-column align-items-center px-0 {{!$status || $status === config('constants.hr.status.new.label') ? 'active text-theme-gray-dark' : 'text-theme-gray-light' }}"
+                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}}>
+                    <div class="position-relative">
+
```
Comment (GauravGusain98) [inline resources/views/hr/application/index.blade.php:55]: remove unwanted line spaces

File: resources/views/hr/application/index.blade.php [hunk 5]
```diff
@@ -46,102 +46,145 @@
...
+                        <span
+                            class=" d-inline-block py-0"
+                            style="font-family: 'Mulish', sans-serif; font-size: 10px;font-weight:700;line-height:12px;position: absolute;top: -10px;right: -1px;">
+                            {{$newApplicationsCount + $inProgressApplicationsCount - $trialProgramCount}}
```
Comment (GauravGusain98) [inline resources/views/hr/application/index.blade.php:59]: Fromat accordingly at all the other places. Add spaces wherever `{{ }}` are used
```suggestion
                            {{ $newApplicationsCount + $inProgressApplicationsCount - $trialProgramCount }}
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please do the requested changes

File: package.json (hunk 3 above)
Comment (tarun9142) [reply package.json]: ok

File: Modules/HR/Resources/views/application/index.blade.php (hunk 2 above)
Comment (tarun9142) [reply Modules/HR/Resources/views/application/index.blade.php]: fixed it

File: .env.example (hunk 1 above)
Comment (tarun9142) [reply .env.example]: ok

File: resources/views/hr/application/index.blade.php (hunk 4 above)
Comment (tarun9142) [reply resources/views/hr/application/index.blade.php:55]: should I merge line 52 and 53 in single line or something else.

Comment (tarun9142) [reply resources/views/hr/application/index.blade.php:55]: ok got it

**Pull request closed without merging**